examples/resources/genesyscloud_telephony_providers_edges_site_outbound_route/apis.md

```diff
@@ -8,8 +8,6 @@
 
 In versions 1.39.0 to 1.48.0 of the provider, this resource was constructed with a different structure. The current version introduces structural changes that are not backwards compatible with those earlier versions.
 
-These changes are currently controlled by a feature flag and are not yet the default behavior. This allows for a phased implementation and thorough testing of this resource before full release.
-
```

Comment:
@bbbco I should remove this, right? I don't see any feature flag use in this package anymore

Reply:
That's correct. Thanks for taking care of this!